config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -28,7 +28,7 @@ presubmits:
         - --provider=gce
         - --runtime-config=batch/v2alpha1=true,admissionregistration.k8s.io/v1alpha1=true
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-csi-serial
-        - --test_args=--ginkgo.focus=CSI.*\[Serial\]|\[Disruptive\] --ginkgo.skip=\[Flaky\]|\[Feature:.+\]|gcePD-external --minStartupPods=8
```

Comment:
@krzyzacy I think it might be time to fix:
- k/k presubmits should all be in the k/k directory
- config/ should have more OWNERs files